packages/shared/src/components/image/Image.tsx

```diff
@@ -21,8 +24,17 @@ const ImageComponent = (
     }
   };
 
-  // eslint-disable-next-line jsx-a11y/alt-text
-  return <img {...props} ref={ref} onError={onError} />;
+  return (
+    <>
+      {isVideoType && (
+        <div className="flex absolute flex-col justify-center my-2 w-full h-40 rounded-xl bg-overlay-float-pepper">
```

Comment:
Did use the colored pepper background as it's dark in design, if I used the themed overflow quarternary that would be white in dark more and pepper in light mode.

Reply:
Let's go with `bg-overlay-tertiary-black` we should just add it to our `overlay` file.
Let me know if you need help with that.

Reply:
If it's a pain you can set it to full height, but might become an issue with the re-layout task as images become flexible.

Reply:
It needed some tweaks but it's fixed now

Reply:
> Let's go with bg-overlay-tertiary-black we should just add it to our overlay file.
Let me know if you need help with that.

For this, trying to understand why should we go with "black" ? any reasons to not use twitter as that is our black color name?

Reply:
This raises issues with eslint "ESLint: bg-overlay-tertiary-black is not a theme color(@dailydotdev/daily-dev-eslint-rules/no-custom-color)" so it would need any changes in the themes as well.

Reply:
This is solved and pushed now, it was cache issue. Thanks Chris!!